tests/loader_layer_tests.cpp

```diff
@@ -1076,6 +1077,23 @@ TEST(MetaLayers, ExplicitMetaLayer) {
         ASSERT_EQ(count, 2U);
         EXPECT_TRUE(check_permutation({regular_layer_name, meta_layer_name}, layer_props));
     }
+    { 
+        const char* env_var_layer_name = "meta_test_layer.json:regular_test_layer.json";
```

Comment:
@charles-lunarg appreciate the response and the info! That makes things more clear to me. Expanding on this test case returning 1 instead of two: I'm specifying two colon separate JSON files in the environment variable which I expect the code to parse into two different loaded layers. Using vkEnumerateInstanceLayerProperties returns 1 while vkEnumerateDeviceLayerProperties returns 0. I will investigate further but let me know if you have any thoughts on that

Reply:
The loader uses the 'platform' separator. If you are running on linux, I believe the path separator is a colon while on windows its a semicolon. The test framework has OS_ENV_VAR_LIST_SEPARATOR for that exact purpose. Which platform are you running on?

Reply:
@charles-lunarg I'm on Fedora Linux! I'll dig in and see if I can figure it out. Thanks again!

Reply:
Another thing to note: The loader test framework 'intercepts' functions related to finding and loading files. This is because the loader will always look in places like `/etc/vulkan/...` but for testing we can't have that. So whats likely happening is that `meta_test_layer.json` is being found and keyed off of because the test framework appends the appropriate path, but `regular_test_layer.json` isn't because its considered to be part of the filename, rather than a separate element.